notebooks/238-dolly-2-instruction-following/238-dolly-2-instruction-following.ipynb

```diff
@@ -0,0 +1,483 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.            "Provide insturction which describes a task bellow or select among predefined examples and model writes a response that completes the request."
</pre><p>Provide <em>instruction</em></p><p><em>bellow-&gt;below</em> </p><p>and model writes response that performs requested task.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1041/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>